### ISSUE TITLE
Error: fs_1.default.rmSync is not a function

### ISSUE THREAD

**Issue by devrajsinghrawat** [matter-labs/zksync#279]
While doing setup for local node using below documentation 
https://github.com/matter-labs/zksync/blob/master/docs/launch.md

at zk init command, system had downloaded few GB files and THROWN an error after the command finished.. Could you suggest of this error has any impact on process further during local node setup 

```
➜  zksync git:(master) zk init
zksync_postgres_1 is up-to-date
zksync_geth_1 is up-to-date
zksync_dev-ticker_1 is up-to-date
zksync_tesseracts_1 is up-to-date
yarn install v1.22.10
[1/4] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.91s.
yarn run v1.22.10
$ yarn workspace zksync prepublish
$ yarn build && rollup -c
$ tsc

build/index.js → dist/main.js...
created dist/main.js in 2.6s
✨  Done in 8.85s.
Keys unpacked
DATABASE_URL = postgres://postgres@localhost/plasma
   Compiling zksync_storage v1.0.0 (/Users/dev-d/1-WorkspaceSlockit/Identity/zksync/core/lib/storage)
    Finished dev [unoptimized + debuginfo] target(s) in 17.41s
Error: fs_1.default.rmSync is not a function
➜  zksync git:(master)
```

**Comment by popzxc** [comment]
Which version of `node.js` do you use?

**Comment by devrajsinghrawat** [comment]
v14.13.1

**Comment by ly0va** [comment]
According to [node.js docs](https://nodejs.org/api/fs.html#fs_fs_rmsync_path_options) rmSync was added in v14.14.0

**Comment by devrajsinghrawat** [comment]
Thanks for the mention .. let me try again with 14.14

**Comment by devrajsinghrawat** [comment]
Thanks @ly0va .. updating node version resolves the issue
